SeriManga: Fix date locale

sinkableShip opened the pull request:
Closes #1580 

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

sinkableShip committed:
SeriManga: fix date locale

sinkableShip committed:
bump extVersionCode

AwkwardPeak7 reviewed (changes requested):
add try catch around date parsing too so such issue doesn't prevent extension from loading chapters

sinkableShip committed:
add try-catch in date parse

sinkableShip committed:
add another description selector

some mangas use the first selector, example:
https://serimangas.com/manga/kara-yonca

and some mangas use the second one, example:
https://serimangas.com/manga/chainsaw-manrenkli

sinkableShip committed:
Fix status selector

AwkwardPeak7 commented on src/tr/serimanga/src/eu/kanade/tachiyomi/extension/tr/serimanga/SeriManga.kt:
```diff
@@ -1,5 +1,6 @@
 package eu.kanade.tachiyomi.extension.tr.serimanga
 
+import android.util.Log
```
```suggestion
```

AwkwardPeak7 commented on src/tr/serimanga/src/eu/kanade/tachiyomi/extension/tr/serimanga/SeriManga.kt:
```diff
@@ -105,12 +109,17 @@ class SeriManga : ParsedHttpSource() {
+        date_upload = try {
+            dateFormat.parse(element.select("span")[2].ownText())?.time ?: 0
+        } catch (e: ParseException) {
+            Log.e("SeriManga", e.toString())
```
```suggestion
```

AwkwardPeak7 reviewed (commented):
no need for log

AwkwardPeak7 committed:
Apply suggestions from code review
